enderio-base/src/main/java/com/enderio/base/EnderIO.java

```diff
@@ -57,6 +69,14 @@ public void gatherData(GatherDataEvent event) {
         }
     }
 
+    public void registerBERS(RegisterRenderers event) {
+        event.registerBlockEntityRenderer(EIOBlockEntities.GRAVEBE.get(), GraveBER::new);
+    }
+    
+    public void inint(FMLClientSetupEvent event) {
+        ItemBlockRenderTypes.setRenderLayer(EIOBlocks.GRAVE.get(), RenderType.cutout());
```

Comment:
Look at the block builders addLayer method. Does this for you, no need for another new event